src/components/Pursuance/PursuancePage.css

```diff
@@ -0,0 +1,22 @@
+.PursuancePage {
+    min-height: 100vh;
```

Comment:
@MartyTheeMartian This looks handy...

Reply:
Yeah, we really need to test on older browsers as well. But this will do it for demo purpose.